models/uniswap/ethereum/uniswap_v3_ethereum_proposals.sql

```diff
@@ -0,0 +1,71 @@
+{{ config(
+    schema = 'uniswap_v3_ethereum',
+    alias = 'proposals',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_time', 'blockchain', 'project', 'version', 'tx_hash'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "uniswap_v3",
+                                \'["soispoke"]\') }}'
+    )
+}}
+
+{% set blockchain = 'ethereum' %}
+{% set project = 'uniswap' %}
+{% set project_version = 'v3' %}
+{% set dao_name = 'DAO: Uniswap' %}
+{% set dao_address = '0x408ed6354d4973f66138c91495f2f2fcbd8724c3' %}
+
+with cte_support as (SELECT 
+        voter,
+        CASE WHEN support = 0 THEN sum(votes/1e18) ELSE 0 END AS votes_against,
+        CASE WHEN support = 1 THEN sum(votes/1e18) ELSE 0 END AS votes_for,
+        CASE WHEN support = 2 THEN sum(votes/1e18) ELSE 0 END AS votes_abstain,
+        proposalId
+FROM {{ source('uniswap_v3_ethereum', 'GovernorBravoDelegate_evt_VoteCast') }}
+GROUP BY support, proposalId, voter),
+
+cte_sum_votes as (
+SELECT COUNT(DISTINCT voter) as number_of_voters,
+       SUM(votes_for) as votes_for, 
+       SUM(votes_against) as votes_against, 
+       SUM(votes_abstain) as votes_abstain, 
+       SUM(votes_for) + SUM(votes_against) + SUM(votes_abstain) as votes_total,
+       proposalId
+from cte_support
+GROUP BY proposalId)
+
+SELECT 
+    '{{blockchain}}' as blockchain,
+    '{{project}}' as project,
+    '{{project_version}}' as version,
+    pcr.evt_block_time as block_time,
+    pcr.evt_tx_hash as tx_hash, -- Proposal Created tx hash
+    '{{dao_name}}' as dao_name,
+    '{{dao_address}}' as dao_address,
+    proposer,
```

Comment:
can we look to add table alias to all columns in select? it helps for quick readability/review

Reply:
same concept to other model

Reply:
Done, if I understood your comment well :)

Reply:
i meant the table alias prefix to the column in select/where/group by/etc -- i.e. ```pcr.votes_far``` so we know which table it comes from. no need to alias the column name, especially when it's the same

Reply:
sorry for confusion!

Reply:
ohhh that makes more sense 👍, done